numpy/core/src/umath/ufunc_object.c

```diff
@@ -3527,7 +3524,6 @@ PyUFunc_Accumulate(PyUFuncObject *ufunc, PyArrayObject *arr, PyArrayObject *out,
     Py_XDECREF(op_dtypes[0]);
 
     NpyIter_Deallocate(iter);
-    NpyIter_Deallocate(iter_inner);
```

Comment:
I honestly have no idea what the inner iterator was ever planned to do.  But it would have been a decade ago anyway.